src/ui/buttons/button-animations/index.js

```diff
@@ -3,7 +3,17 @@
 import { type Personalization } from '../props';
 
 import { setupDivideLogoAnimation } from './divide-logo-animation';
-import { type ButtonAnimationOutputParams } from './types';
+import { setupFadeOutLogoAndShowLabelAnimation } from './fadeout-logo-show-label-text';
+import type { ButtonAnimationIds, ButtonAnimationOutputParams } from './types';
+
+
+function setupAnimation(animationId : string, animationLabelText : string) : ButtonAnimationOutputParams | null {
+    const animationIds : ButtonAnimationIds = {
```

Comment:
Please add the static button design ID here as well

Reply:
done